tmt/export.py

```diff
@@ -75,6 +75,11 @@ def export_to_nitrate(test, create, general):
     # Components
     # First remove any components that are already there
     nitrate_case.components.clear()
+    if general:
+        # Remove also all general plans linked to testcase
+        for nitrate_plan in nitrate_case.testplans:
+            if is_general_component_plan(nitrate_plan):
```

Comment:
Is it worth of introducing a new function if the condition could be simple as this?

```suggestion
            if nitrate_plan.type.name == 'General':
```

Plus the function is only used once.

Reply:
I've expected there more code at the beginning so that I've decide to do that by separate function. But you are right, it is so easy, that it could be inline the code